testsuite/MDAnalysisTests/__init__.py

```diff
@@ -98,11 +98,6 @@
 
 # keep in sync with RELEASE in setup.py
 __version__ = "0.19.1-dev"
-try:
```

Comment:
Since MDAnalysis is a dependency of MDAnalysisTests, we could have `__authors__ = MDAnalysis.__authors__`.

Reply:
@jbarnoud I tried this a few commits back, it messed up duecredit, and I think it also messes with coverage:
 https://github.com/MDAnalysis/mdanalysis/blob/b52e961d992abd43911e1cfb97b2468d92670e92/testsuite/MDAnalysisTests/__init__.py#L103